build-tools/packages/build-cli/src/library/git.ts

```diff
@@ -43,6 +43,45 @@ function getVersionFromTag(tag: string): string | undefined {
 	return ver.version;
 }
 
+export async function getAllPackageVersions(
+	releaseGroupOrPackage: string,
```

Comment:
Wherever possible, prefer APIs that take IReleaseGroups or IPackages. If absolutely necessary, you can also take `IPackageName | IReleaseGroupName` - but try to avoid passing strings around as standins for packages or release groups. "Convert" them into proper release groups/packages early and then pass those around.